### DISCUSSION TITLE
Make the template RTL

### DISCUSSION THREAD

**Discussion by ahmedDawelbait** [orchidsoftware/platform#2097] [Q&A] [answered]
Hello everyone, I currently working on dashboard using orchid but i can't change direction of template to be RTL if anyone tried it help me please.
Thank you in advance

**Comment by NacerKH** [comment, marked as answer]
hello @ahmedDawelbait  
just change Application Locale Configuration to ar, so go to config/app.php  and change locale to "ar"
